cypress.config.js

```diff
@@ -5,41 +5,51 @@ const { addCucumberPreprocessorPlugin } = require('@badeball/cypress-cucumber-pr
 const allureWriter = require('@shelex/cypress-allure-plugin/writer');
 const webpack = require('@cypress/webpack-preprocessor');
 
-const { ENV } = process.env;
-const DEV = 'dev';
-const QA = 'qa';
+const DEV = 'DEV';
+const QA = 'QA';
 const PROD = 'prod';
-const currentEnv = ENV || QA;
+const currentEnv = QA;
 
 const envs = {
   currentEnv,
   DEV,
   QA,
   PROD,
-}
+};
+
+/**
+ *
```

Comment:
You might want to add a description here ;-)

Reply:
You may think that, but I want it to be a surprise. 🥳